x-pack/plugins/security_solution/public/detections/components/osquery/osquery_flyout.tsx

```diff
@@ -57,13 +55,13 @@ export const OsqueryFlyoutComponent: React.FC<OsqueryFlyoutProps> = ({ agentId,
         queryMatch: {
           field,
           value,
-          operator: ':',
+          operator: ':' as const,
         },
-      };
+      }));
 
       return getAddToTimelineButton({
-        dataProvider: providerA,
-        field: value,
+        dataProvider: providers,
```

Comment:
This change is required for us to add multiple actions to time at once. Is this a proper approach? 
The functionality itself is not used yet, but I am hoping to use it still for 8.5